File: app/src/main/java/com/lazysoul/kotlinwithandroid/datas/Todo.java
package com.lazysoul.kotlinwithandroid.datas;

/**
 * Created by Lazysoul on 2017. 7. 9..
 */

public class Todo {

    private int id;

    private String body;

    private boolean isChecked;

    private boolean isFixed = false;

    public int getId() {
        return id;
    }

    public void setId(int id) {
        this.id = id;
    }

    public String getBody() {
        return body;
    }

    public void setBody(String body) {
        this.body = body;
    }

    public boolean isChecked() {
        return isChecked;
    }

    public void setChecked(boolean checked) {
        isChecked = checked;
    }

    public boolean isFixed() {
        return isFixed;
    }

    public void setFixed(boolean fixed) {
        isFixed = fixed;
    }
}

File: app/src/main/kotlin/com/lazysoul/kotlinwithandroid/datas/Todo.kt
package com.lazysoul.kotlinwithandroid.datas

import io.realm.RealmObject
import io.realm.annotations.Ignore
import io.realm.annotations.PrimaryKey
import java.util.Date

/**
 * Created by Lazysoul on 2017. 7. 24..
 */
open class Todo(@PrimaryKey var id: Int = -1, var body: String = "",
        var isChecked: Boolean = false,
        var createdAt: Date? = null, @Ignore var isFixed: Boolean = false) : RealmObject()